src/simplexinv.jl

```diff
@@ -60,43 +65,49 @@ function simplexinv(c, A, b, IB=0, invB=0; max_iter = 4000)
 
   x = zeros(n)
   if !artificial
-    x[IB] = xB
+    x[𝔹] = xB
     z = dot(c, x)
   else
-    if dot(xB, c[IB]) > 0
-      status = :Infeasible
-      I = find(IB .<= n - m)
-      x[I] = xB[I]
-      z = dot(co, x)
-    elseif maximum(IB) > n # check for artificial variables in basis
-      deleteat!(IN, find(IN .> n))
+    if dot(xB, c[𝔹]) > eps(Float64)

```

Comment:
This should be further analyzed. Does it happen? Is `c` being modified? Is `xB` close to 0 when it should be zero? Is there an example? Please open an issue.